src/ve-app/pane-left/left-pane.component.ts

```diff
@@ -413,13 +413,16 @@ class LeftPaneController implements angular.IComponentController {
                 }
                 this.eventSvc.$broadcast<veAppEvents.elementSelectedData>('element.selected', data)
             }
-            void this.$state.go(
-                'main.project.ref.view.present.' + (this.$uiRouterGlobals.params as ParamsObject).display,
-                {
-                    viewId,
-                    search: undefined,
-                }
-            )
+            if (this.rootScopeSvc.veFullDocMode()) {
+                this.eventSvc.$broadcast<string>('view.scroll', viewId)
+            } else {
+                void this.$state.go(
+                    'main.project.ref.view.present.' + (this.$uiRouterGlobals.params as ParamsObject).display,
+                    {
+                        viewId,
+                        search: undefined,
+                    })
+            }
```

Comment:
If we are already in FullDocView a transition here should only change the ViewId parameter. If we don't do a transition here it might be hard to get accurate links for users to copy with the right ?viewId. Is the transition doing something funny?

Reply:
the transition reloads the center pane, so full doc compoent will keep reloading which we dont' want